apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -818,53 +819,68 @@ const DanceAiModal: React.FunctionComponent = () => {
         )}
 
         <div id="buttons-area-top" className={moduleStyles.buttonsAreaTop}>
-          {mode === Mode.RESULTS && (
-            <div>
-              <Button
-                id="explanation-button"
-                onClick={handleExplanationClick}
-                color={Button.ButtonColor.neutralDark}
-                className={moduleStyles.button}
-              >
-                <i className="fa fa-bar-chart" />
-              </Button>
-            </div>
-          )}
+          <div id="buttons-area-top-left">
+            <ModalButton
+              currentMode={mode}
+              showFor={[Mode.RESULTS, Mode.EXPLANATION]}
+              disableFor={[Mode.EXPLANATION]}
+              id="start-over-button"
+              onClick={handleStartOverClick}
+              color={Button.ButtonColor.neutralDark}
+              className={moduleStyles.button}
+              aria-label={i18n.danceAiModalStartOverButton()}
+              icon="fast-backward"
+            />
+          </div>
+          <div id="buttons-area-top-right">
+            <ModalButton
+              currentMode={mode}
+              showFor={[Mode.RESULTS]}
+              id="explanation-button"
+              onClick={handleExplanationClick}
+              color={Button.ButtonColor.neutralDark}
+              className={moduleStyles.button}
+              aria-label={i18n.danceAiModalExplanationButton()}
```

Comment:
@breville just confirming this was the intended label for this button

Reply:
Yes, I believe so.